pkg/ovs/ovn-nb-address_set.go

```diff
@@ -51,6 +52,9 @@ func (c *ovnClient) AddressSetUpdateAddress(asName string, addresses ...string)
 		return fmt.Errorf("get address set %s: %v", asName, err)
 	}
 
+	// update will failed when slice contains duplicate elements
+	addresses = util.UniqString(addresses)
+
```

Comment:
```suggestion
	// update will failed when slice contains duplicate elements
	addresses = util.UniqString(addresses)

	// format CIDR to keep addresses the same in both nb and sb 
	for i, addr := range addresses {
		if strings.ContainsRune(addr, '/') {
			_, ipNet, err := net.ParseCIDR(addr)
			if err != nil {
				klog.Warningf("failed to parse CIDR %q: %v", addr, err)
				continue
			}
			addresses[i] = ipNet.String()
		}
	}
```

Fixes the following occasional e2e test failure:

```txt
[FAIL] [sig-network] Netpol NetworkPolicy between server and client [It] should ensure an IP overlapping both IPBlock.CIDR and IPBlock.Except is allowed [Feature:NetworkPolicy]
```